src/applications/personalization/profile/tests/e2e/personal-and-contact-info/modals.cypress.spec.js

```diff
@@ -180,3 +181,37 @@ describe('Modals on the personal information and content page after editing', ()
     cy.findByLabelText(/email address/i);
   });
 });
+
+describe('Modals on the personal information and content page when they error', () => {
+  it('should allow the ability to reopen the edit modal when the transaction completes', () => {
```

Comment:
this `it` description doesn't seem right

Reply:
wow sorry about that. I should have noticed that.